exchange/events.go

```diff
@@ -62,7 +62,11 @@ func (ev *eventTracking) modifyBidVAST(pbsBid *pbsOrtbBid, bidderName openrtb_ex
 		return
 	}
 	vastXML := makeVAST(bid)
```

Comment:
I understand that unit tests for this function already don't exist but do you think we can incorporate them as part of this PR if it's not a lot of effort?

Reply:
There is no specific unit test for this function, but it's covered by these test cases as a part of `TestExchangeJSON`: 
exchange/exchangetest/events-vast-account-off-request-on.json
exchange/exchangetest/events-vast-account-on-request-off.json
There are some other functions like this: `makeEventURL`, `modifyBidVAST`, `ModifyVastXmlString`

Reply:
I see. Thanks! Honestly, it'd be good to have some unit test coverage for these as well but that's out of scope for this PR